### PR TITLE
chore: Upgrade React Router to v3

### PR DESCRIPTION
**Pull request by lynnagara**
Required for React 16

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: package.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "react-icon-base": "^2.0.4",
     "react-lazy-load": "3.0.10",
     "react-mentions": "^1.0.0",
-    "react-router": "2.8.1",
+    "react-router": "3.0.2",
```
Comment (billyvg) [inline package.json]: Can we use the latest version that includes react 16 support (3.2.0)?

**Review by billyvg — COMMENTED** [review]
Let's do some manual testing on staging before we deploy this

File: package.json (hunk 1 above)
Comment (lynnagara) [reply package.json]: Yes!

**Comment by MaxBittker** [comment]
interesting percy regression? or is there no way RR caused this?

**Comment by lynnagara** [comment]
Hmm weird, I can't see how that would be caused by these changes though

**Comment by billyvg** [comment]
Yeah it's unrelated to this PR

**Comment by lynnagara** [comment]
@MaxBittker @billyvg Deployed to staging, looks fine to me

**Review by MaxBittker — APPROVED** [review]
🎉 🥉    𓀬

**Pull request merged**